include/camp/camp.hpp

```diff
@@ -151,19 +151,23 @@ namespace test
 }
 #endif
 
+
+/**
+ * @brief Get the index of the first instance of T in L
```

Comment:
Woooo, documentation 😄

Reply:
Yup.  Much more is needed, but it seemed like a place to start.  CAMP just got its release today, so I'll probably be taking a pass over all of that in the near future.